testdata/repos/standalone/EbuildUnquotedVariablesCheck/EbuildUnquotedVariable/EbuildUnquotedVariable-0.ebuild

```diff
@@ -0,0 +1,40 @@
+EAPI=8
+DESCRIPTION="Ebuild with variables that must be quoted"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
+SLOT="0"
+LICENSE="BSD"
+S=${WORKDIR}/${PV} # ok
+
+PATCHES=(
+	${FILESDIR}/foo.patch # FAIL
+	"${FILESDIR}"/foo.patch # ok
+	"${FILESDIR}/foo.patch" # ok
+)
+
+src_prepare() {
+	if [[ -z ${S} ]]; then  # ok
+		:
+	fi
+
+	if has "foo" ${FILESDIR} ; then # FAIL
+		:
+	fi
+
+	local t=${T} # ok
+	local t=${T}/t # ok
+	emake CC=${T} # FAIL
+
+	local var=( TMP=${T} ) # FAIL
+
+	cat > "${T}"/somefile <<- EOF || die
+		PATH="${EPREFIX}${INSTALLDIR}/bin"
+	EOF
+	doenvd "${T}"/10stuffit # ok
+
+	echo "InitiatorName=$(${WORKDIR}/usr/sbin/iscsi-iname)" # FAIL
+	einfo ${WORKDIR} # ok
+
+	if grep -qs '^ *sshd *:' "${WORKDIR}"/etc/hosts.{allow,deny} ; then # ok
+		ewarn "something something ${WORKDIR} here"
+	fi
```

Comment:
Let's add here some extra examples:

1. `cat < ${T}` - should fail
2. `cat <<< ${T}` - should pass (I think, at least it looks like it works in bash)
3. `cat >> ${T}` - should fail

Reply:
useless use of cat? :P

I'll add them.

Reply:
They check quoting for `<` and `>>` operators.

Reply:
Added.